sql/catalyst/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Relation.scala

```diff
@@ -167,6 +173,17 @@ case class StreamingDataSourceV2Relation(
     case _ =>
       Statistics(sizeInBytes = conf.defaultSizeInBytes)
   }
+
+  private val stringArgsVal: Seq[Any] = {
+    val tableQualifier = (catalog, identifier) match {
```

Comment:
Printing out `catalog` and `identifier` separately does not seem to give a good output, hence I changed to produce a single output for composite fields `catalog` and `identifier` whenever they are provided altogether in the logical node (only on streaming code path).